api-matlab/wrapper_physiobatch.m

```diff
@@ -0,0 +1,111 @@
+function matlabbatch = wrapper_physiobatch(data_vendor, out_dir, default_path)
+    spm_jobman('initcfg');
+    global defaults
+    defaults.stats.maxmem   = 2^30;
+    matlabbatch = {};
+    
+    matlabbatch{1}.spm.tools.physio.save_dir = {[out_dir filesep 'physio_dir']};
+    matlabbatch{1}.spm.tools.physio.log_files.vendor = data_vendor;
+    
+    % EXAMPLES IN C:\Users\nwiedemann\Downloads\tapas\tapas\misc\example\PhysIO
+    
+    % different vendor options require different file input
+    if strcmp(data_vendor, 'BIDS')
+        [tsv_filename, ~] = spm_select(1,'.*.tsv.*','Specify tsv/tsv.gz file with ECG data',{}, default_path);
+        % tsv_filename = input(['For BIDS, you only need to specify the tsv/tsv.gz file containing 3 columns without header.' ...
+         % 'There must be a json with the same file name in the same folder containing metainformation'], 's');
+        matlabbatch{1}.spm.tools.physio.log_files.cardiac = {tsv_filename}; 
+        matlabbatch{1}.spm.tools.physio.log_files.respiration = {''}; 
+        matlabbatch{1}.spm.tools.physio.log_files.scan_timing = {''};
+    elseif strcmp(data_vendor, 'Philips')
+        [scanphyslog, ~] = spm_select(1,'.*.log','Specify SCANPHYSLOG.log file',{}, default_path);
```

Comment:
For some reason the SPM select box takes multiple seconds to open up on my system. @NinaWie do you experience the same behaviour? Just trying to see if it's just my system (which typically doesn't have this type of delay)

Reply:
It also takes a while on my system and actually the spm select box is not very user friendly anyways. I will change it to a matab file select object.